src/signalo/core/models.py

```diff
@@ -22,7 +27,11 @@ def _serialized(self):
         return f'{{"id": "{str(self.id)}", "type": "Feature", "geometry": {self.geom.geojson}, "properties": {{"name": "{self.name}"}}}}'
 
 
-@register_oapif_viewset()
+@register_oapif_viewset(
+    custom_viewset_attrs={
+        "permission_classes": (permissions.IsAuthenticatedOrReadOnly,)
+    }
```

Comment:
Same (Leave to default, which better matches the use case we want to demonstrate.)